sphinx_gherkindoc/cli.py

```diff
@@ -36,15 +50,24 @@ def process_args(
 
     non_empty_dirs: Set[pathlib.Path] = set()
 
+    get_url_from_tag = None  # noqa: E731
```

Comment:
So I know this was there on line 40 before, but why does it need noqa?
https://www.flake8rules.com/rules/E731.html
You're not assigning a lambda to it, so I really don't know why it's mad?

Reply:
Yeah I changed this from a lambda to None, but I didn't think about the noqa comment. It should be fine to remove. If not I'll let you know.